app/Services/OperatingSystem.php

```diff
@@ -15,6 +15,7 @@ class OperatingSystem {
      * @see http://0pointer.de/blog/projects/os-release
      */
     public const OS_RELEASE = '/etc/os-release';
+    public const WLS2_ENV   = 'WSL_DISTRO_NAME';
```

Comment:
Is that right name variable? `WLS2_ENV` -> `WSL2_ENV`

Reply:
Renamed in https://github.com/moderntribe/square1-global-docker/pull/108/commits/f90d93a36306d8583bf2e04514ad90c57512acb8